experiments/integrated/fluxnet/data_tools.jl

```diff
@@ -105,29 +125,44 @@ the mean value per the QC flag.
  - Checks for missing data and replaces with the mean value
 - Returns an `absent` status if the column is not present.
 """
+
 function filter_column(driver_data::Matrix, column_name::String, units::String)
     # Check that the data exists and read it in if so
     col_dat, status = column_status(driver_data, column_name)
     # Set missing data threshold above which column is treated as absent
     missing_threshold = 0.5
     # Set poor quality threshold above which the column undergoes no replacement using the quality flag
-    quality_threshold = 0.5
+    quality_threshold = 50 # if more than 50% of the data in a site is poor quality
```

Comment:
@Espeer5 Hey Ed, Mitra is wondering where the 0.5 (missing and quality thresholds) is used? and if it is okay to be 50

Reply:
It's used only right below that where the percentages are compared to determine when a column should be marked as missing or not. 50% should be fine, it should be flexible.